tests/functional/data_classes/test_amazon_mq.py

```diff
@@ -47,6 +48,7 @@ def test_rabbit_mq_event():
     assert message.data is not None
     assert message.decoded_data is not None
     assert message.json_data["timeout"] == 0
+    assert message.json_data["timeout"] == 0  # cached lookup
```

Comment:
As the data is the same, what are we testing here? since that would've been the same prior to this change.

Unless you're trying to do a timer test to verify deserialization access time is different when accessing one more time